resources/integration_test/tests/dbmd_variable_result_sets_data_methods.yml

```diff
@@ -66,7 +66,7 @@ tests:
 
   - description: getTables_null_parameters_returns_all_tables
     db: integration_test
-    meta_function: [ getTables, null, null, null, null ]
+    meta_function: [getTables, null, null, null, null]
```

Comment:
Changes in these integration test files are due to adding a new `uuid` collection.